protocols/zerolend/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ZeroLend",
+  "icon": "logo.png",
+  "metadata": {
+    "pt": [
+      {
+        "chainId": 1,
+        "address": "0x7bAf258049cc8B9A78097723dc19a8b103D4098F",
```

Comment:
Notice this is for PTrsETH Sept which has already matured. Pls remove this market